_getting-started/concepts-system-overview-guide.md

```diff
@@ -0,0 +1,394 @@
+
+              The [Import API]({{ link.import-api.getting-started | prepend: site.baseurl }}) is a [Clojure](https://clojure.org/){:target="new"} web service that accepts JSON and [Transit](https://github.com/cognitect/transit-format){:target="new"}, either in large batches or at a point in time. The Import API performs a validation check on the data before writing it to a central Apache Kafka queue.
+
+              If the data fails validation or another critical error occurs, the extraction will fail and trigger an [in-app and email notification]({{ link.account.notification-settings | prepend: site.baseurl }}). The error can also be viewed in an integration's [Extraction Logs]({{ link.replication.extraction-logs | prepend: site.baseurl }}) tab.
+
+              ---
+
+      - title: "Preparing"
+        anchor: "system-architecture--preparing"
+        content: |
+          The second phase in the replication process is called **Preparing**. During this phase, the extracted data is replicated internally for redundancy, batched, and readied for loading.
+
+          The Preparing phase includes:
+
+          {% for sub-subsection in subsection.sub-subsections %}
+          - [{{ sub-subsection.title }}](#{{ sub-subsection.anchor }})
+          {% endfor %}
+
+        sub-subsections:
+          - title: "Step 3: The Kafka queue"
+            anchor: "system-architecture--kafka-queue"
+            content: |
+              Data sent from the Import API is immediately persisted to Kafka, using Amazon S3 as a fallback. The Kafka queue is used to ensure Stitch meets its most important service-level target: Don't lose data.
+
+              During this step, Stitch replicates our Kafka cluster across three different data centers. Each data point in the extracted data must be written to two data centers before it's accepted. Should the write fail, Stitch will continue to try until it's successful.
+
+              Data is stored encrypted, and deleted after no more than seven days.
+
+          - title: "Step 4: The Streamery"
+            anchor: "system-architecture--streamery"
+            content: |
+              Next, data is moved from Kafka into what is called the Streamery. A low-latency, multithreaded Clojure application, the Streamery's goal is to maximize throughput while guarding against data loss or data leaking between data sets.
+
+              During this step, data is written to files on Amazon S3 in batches. Batches are separated by Stitch client, integration, and the tables the data is destined for.
+
+              After reaching either a memory limit or an amount of time elapsed since the last batch, the Streamery cuts a batch.
+
+          - title: "Step 5: The Spool"
```

Comment:
I think we should remove this step, it's an implementation detail that shouldn't really matter